### ISSUE TITLE
Upgrade to 0.11.0

### ISSUE THREAD

**Issue by tomlankhorst** [ofnode/of#25]
I wonder if there are there any plans to work on this? 
No promises from my side yet, but would you be open to PRs?

**Comment by MatthewScholefield** [comment]
> I wonder if there are there any plans to work on this?

I don't believe so simply because no one is actively working on the project. But if so, @avilleret would have an answer. There is a [0.10 branch](https://github.com/ofnode/of/tree/0.10) that might be of relevance.

> No promises from my side yet, but would you be open to PRs?

Absolutely, I'd be happy to merge any PRs with improvements. (Note: I'm not the author of this repo; just a previous contributor with repo access)

**Comment by avilleret** [comment]
I'm not working on this repo anymore since I'm not working with openFrameworks anymore, but yes PRs are welcome.
